components/base/base.go

```diff
@@ -42,22 +42,73 @@ type Base interface {
 	// MoveStraight moves the robot straight a given distance at a given speed.
 	// If a distance or speed of zero is given, the base will stop.
 	// This method blocks until completed or cancelled
+	// 
```

Comment:
Test is failing because of trailing white space. You can adjust the settings in VS code to remove trailing whitespace on save. Let me know if you need help setting that up!
```suggestion
	//
```